adagrams/game.py

```diff
@@ -80,21 +80,25 @@ def get_highest_word_score(word_list):
 
 # get highest value or add values to ties list
     for word_item, points in score_tracker.items():
+
         if points > winning_score:
             winning_score = points
             winning_word = word_item
+            
             if len(ties) > 0:
-                ties.clear
+                ties.clear()
```

Comment:
👍 Yep. We needed to call the method. This resets the ties that we're tracking to only reflect the new highest score.

Notice that Python didn't give a squeek about us referencing a method without calling it. It was perfectly content to let us retrieve the method reference, and then basically throw it away without using it. So we need to be really careful of this in general. Adding additional tests (such as for the scenario I pointed out in my previous comment) that depend on the function being properly called can help us notice this.